### ISSUE TITLE
Pause and Wait keys in dial dialog

### ISSUE THREAD

**Issue by stephanschuler** [64characters/Telephone#348]
Hey there.

This is another feature request, obviously :).

User story:

> As a regular participant in telephone conferences with pin code protection, I want to not only put the phone number into the dial dialog but the pin code as well.

Both, Android and iOS support special characters `,` and `p` to delay further keystrokes for a second as well as `;` and `w` to delay further key strokes until the user confirms.

Here's a good descriptions of the feature.

http://allgaierconsulting.com/techtalk/2014/8/1/why-and-how-to-insert-a-pause-or-wait-key-on-your-iphone
http://www.androidpolice.com/2010/05/10/how-to-add-hard-wait-and-soft-2-3-sec-pauses-to-your-android-contacts/

I have several entries in my address book making use of this feature. Currently those special characters are just stripped before dialing. Depending on the system receiving my call, this results either in just not handling the additional digits at all (which is can easily deal with) or rejecting the call with "There is no such number" or "All lines are busy right now" (usually when the conferencing system is Skype for Business).
Would be awesome if Telephone supported this feature :).

Regards,
Stephan.

**Comment by eofster** [comment]
Agreed, this is an important feature that finally needs to be implemented. We also have it here: #67.
